pages/explore/components/ExploreMenu.tsx

```diff
@@ -61,7 +61,6 @@ const styles = StyleSheet.create({
   inactiveCategory: {
     padding: 5,
     paddingVertical: 10,
```

Comment:
this vertical padding is too big for some screen sizes hence the text in the button gets cut off sometimes. 
I have reduced this to 2 in the latest commit.